pallets/parachain-staking/src/lib.rs

```diff
@@ -99,13 +101,18 @@ pub mod pallet {
 	pub type BalanceOf<T> =
 		<<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
 
+	pub const COLLATOR_LOCK_IDENTIFIER: [u8; 8] = *b"ColStake";
```

Comment:
Please use standard naming for locks with our other locks (like democracy)

Reply:
ex: `stakingcol` & `stakingdel`

Reply:
I adjusted this to `stkngcol` and `stkngdel` in 69dbe7d which is a little more consistent. I'm open to suggestions on alternatives, but it has to be exactly 8 bytes.